### ISSUE TITLE
Status bar problem 

### ISSUE THREAD

**Issue by jonathannaguin** [kirbyt/KTPhotoBrowser#25]
Hi,
I have a problem when a send my application to background. When i came back to foreground its display like http://piczasso.com/i/a0owu.jpg

Step to play the bug on iPad simulator with the Sample:
1- Wait until chrome is hiden.
2- Press Home button to send the Sample to background.
3- Press again in Sample icon to launch Sample.
4- Tap image.

Thank you

**Comment by andywei** [comment]
Yes, I find the same problem on iPhone (OS4.3) + KTPhotoBrowser (March 25, 2011).

**Comment by andywei** [comment]
I've fixed this defect by adding one line of code in KTPhotoScrollViewController.m. Please check below:
- (void)toggleChrome:(BOOL)hide 
  {
  ...
  UINavigationBar *navbar = [[self navigationController] navigationBar];
  [navbar setAlpha:alpha];
  
  //Add this:  
  [[self navigationController] setNavigationBarHidden:hide animated:NO];
  //Add this end
  
     [toolbar_ setAlpha:alpha];
  
     ...
  }

**Comment by kissfro** [comment]
I tried that out Andy it doesn't seem to work for me. Did you do anything else different?
